packages/opentelemetry-semantic-conventions-ai/opentelemetry/semconv_ai/__init__.py

```diff
@@ -236,3 +236,26 @@ class TraceloopSpanKindValues(Enum):
     AGENT = "agent"
     TOOL = "tool"
     UNKNOWN = "unknown"
+
+class MetricUtils:
+
+    def basic_metrics(meter):
```

Comment:
Let's make it a typed function?

Reply:
Sure, will add that

Reply:
Added, can you please check if it looks good